ecaudit/src/main/java/com/ericsson/bss/cassandra/ecaudit/auth/WrappingAuditAuthenticator.java

```diff
@@ -140,5 +160,17 @@ public AuthenticatedUser getAuthenticatedUser() throws AuthenticationException
                 throw e;
             }
         }
+
+        @Override
+        public String getUser()
```

Comment:
So, this is why it feels wrong to implement the IAuditAuthenticator interface in this class. Is there a valid scenario where this method should be called?

Reply:
See above